cmd/lncli/commands.go

```diff
@@ -3363,7 +3363,8 @@ var updateChannelPolicyCommand = cli.Command{
 	Category: "Channels",
 	Usage: "Update the channel policy for all channels, or a single " +
 		"channel.",
-	ArgsUsage: "base_fee_msat fee_rate time_lock_delta [channel_point]",
+	ArgsUsage: "base_fee_msat fee_rate time_lock_delta max_htlc_msat " +
```

Comment:
So now we'll only allow `max_htlc_msat` to be set with the flag explicitly, but this seems to indicate that you can specify it without the flag?